x/wasm/keeper/msg_server_integration_test.go

```diff
@@ -168,7 +168,14 @@ func TestAddCodeUploadParamsAddresses(t *testing.T) {
 			},
 			uploadConfig:    types.AccessTypeAnyOfAddresses.With(myAddress),
 			expUploadConfig: types.AccessTypeAnyOfAddresses.With(myAddress, otherAddr),
-			expErr:          false,
+		},
+		"add existing address": {
```

Comment:
I add covered it in: "authority cannot add existing addresses when permission is any of addresses". We can remove one of them.